sdk/core/core-amqp/review/core-amqp.api.md

```diff
@@ -28,9 +28,43 @@ export { AccessToken }
 
 export { AmqpMessage }
 
-export { AmqpMessageHeader }
+// @public
+export interface AmqpMessageHeader {
+    deliveryCount?: number;
+    durable?: boolean;
+    firstAcquirer?: boolean;
+    priority?: number;
+    timeToLive?: number;
```

Comment:
Since this interface needs to be as close to the AMQP spec as possible, should `timetoLive` be `ttl`?

Reply:
Do we not want to be in sync with other languages?